src/NHSD.BuyingCatalogue.Ordering.Services/OrderService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using NHSD.BuyingCatalogue.Ordering.Contracts;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+using NHSD.BuyingCatalogue.Ordering.Persistence.Data;
+
+namespace NHSD.BuyingCatalogue.Ordering.Services
+{
+    public class OrderService : IOrderService
+    {
+        private readonly ApplicationDbContext context;
+
+#pragma warning disable CS3001 // Argument type is not CLS-compliant
+        public OrderService(ApplicationDbContext context)
+#pragma warning restore CS3001 // Argument type is not CLS-compliant
```

Comment:
We should be able to remove the `#pragma warning` now that you have marked the assembly as `CLSCompliant: false`.

Reply:
Removed comments, not needed with assembly as `CLSCompliant: false`